aspnet/migration/rc1-to-rtm.rst

```diff
@@ -0,0 +1,483 @@
+
+The following action result types have also been renamed:
+
+=============================================  =============================================
+ASP.NET 5 RC1                                        ASP.NET Core 1.0
+=============================================  =============================================
+Microsoft.AspNet.Mvc.HttpOkObjectResult        Microsoft.AspNetCore.Mvc.OkObjectResult
+Microsoft.AspNet.Mvc.HttpOkResult              Microsoft.AspNetCore.Mvc.OkResult
+Microsoft.AspNet.Mvc.HttpNotFoundObjectResult  Microsoft.AspNetCore.Mvc.NotFoundObjectResult
+Microsoft.AspNet.Mvc.HttpNotFoundResult        Microsoft.AspNetCore.Mvc.NotFoundResult
+Microsoft.AspNet.Mvc.HttpStatusCodeResult      Microsoft.AspNetCore.Mvc.StatusCodeResult
+Microsoft.AspNet.Mvc.HttpUnauthorizedResult    Microsoft.AspNetCore.Mvc.UnauthorizedResult
+=============================================  =============================================
+
+Changes in views
+^^^^^^^^^^^^^^^^
+
+Views now support relative paths.
+
+The Validation Summary Tag Helper has changed. Change:
+
+.. code-block:: html
+
+  <div asp-validation-summary="ValidationSummary.All"></div>
+
+To:
+
+.. code-block:: html
+
+  <div asp-validation-summary="All"></div>
+
+Changes in ViewComponents
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The sync APIs have been removed.
+
+To reduce ambiguity in ViewComponent method selection, we've modified the selection to only allow exactly one ``Invoke()`` or ``InvokeAsync()`` per ViewComponent.
+``Component.Render()``, ``Component.RenderAsync()``, and ``Component.Invoke()`` have been removed.
+
+``InvokeAsync()`` now takes an anonynmous object instead of separate parameters. To use the view component, call ``@Component.InvokeAsync("Name of view component", <parameters>)`` from a view. The parameters will be passed to the ``InvokeAsync()`` method. The following example demonstrates the ``InvokeAsync()`` method call with two parameters:
+
+ASP.NET 5 RC1:
+
+.. code-block:: c#
+
+  @Component.InvokeAsync("Test", "MyName", 15)
+
+ASP.NET Core 1.0:
+
+.. code-block:: c#
+
+  @Component.InvokeAsync("Test", new { name = "MyName", age = 15 })
+  @Component.InvokeAsync("Test", new Dictionary<string, object> { ["name"] = "MyName", ["age"] = 15 })
+  @Component.InvokeAsync<TestViewComponent>(new { name = "MyName", age = 15})
+
+Updated controller discovery rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are changes that simplify controller discovery:
+
+The new :dn:class:`~Microsoft.AspNetCore.Mvc.ControllerAttribute` be used to mark a class and their subclasses as controllers. A classe whose name doesn't end in ``Controller`` and derives from a base class that ends in ``Controller`` is no longer considered a controller. In this scenario, :dn:class:`~Microsoft.AspNetCore.Mvc.ControllerAttribute` must be applied to the derived class itself or to the base class.
+
+A type is not considered a controller if all the following conditions apply:
```

Comment:
>A type is not considered a controller if all the following conditions apply: 

@rynowak  I'm removing not, I think this is backwards. 
A type is ~~not~~ considered a controller if all the following conditions apply: